lib/libutee/include/tee_api.h

```diff
@@ -33,6 +33,9 @@
 #include <compiler.h>
 #include <tee_api_defines.h>
 #include <tee_api_types.h>
+#if defined(CFG_TEE_PANIC_DEBUG)
```

Comment:
This flag is here mainly to be able to have `TEE_Panic()` optionally defined as a function to still be GP compliant. 
It doesn't have much to do with log levels in my opinion.